build-parent/pom.xml

```diff
@@ -608,7 +608,6 @@
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
                             <release>8</release>
-                            <parameters>true</parameters>
```

Comment:
Note: This is already set in the general `<pluginManagement>` further up.

Reply:
Why do we have the `<release>8</release>` in a specific profile?

Reply:
Last time I checked, the plugin or better to say the compiler throws an error on JDK8 because it is a JDK9+ feature.
Like you said, if something makes life a bit less miserable, it's JDK 11 only. 😉

Reply:
PS: The only way I know to enforce Java 8 API-compliance (or even 7, 6 etc.) with JDK 8 or below is https://github.com/mojohaus/animal-sniffer

Reply:
Yeah, OK, makes sense, then!